examples/simple_service.rs

```diff
@@ -20,7 +25,7 @@ const DEFAULT_MAX_ROWS: usize = 100;
 pub async fn odata_service_handler(
     axum::extract::State(query_ctx): axum::extract::State<SessionContext>,
     host: axum::extract::Host,
-) -> axum::response::Response<String> {
+) -> AxumResult<Response<String>> {
```

Comment:
I don't see the point of wrapping everything into `AxumResult`, I think we can simply return `std::result::Result<Response<T>, Error>` - this will work because `Error` implements `IntoResult` and eliminate one extra layer of type conversion allowing us to use `?` more.

Reply:
Hey, I have no experience with axum library, so I have no opinion here. I just wanted to use `?` within the handlers.